src/qt/rpcconsole.cpp

```diff
@@ -1268,22 +1268,23 @@ void RPCConsole::showOrHideBanTableIfRequired()
 
 void RPCConsole::setTabFocus(enum TabTypes tabType)
 {
-    ui->tabWidget->setCurrentIndex(tabType);
+    ui->tabWidget->setCurrentIndex(int(tabType));
 }
 
 QString RPCConsole::tabTitle(TabTypes tab_type) const
 {
-    return ui->tabWidget->tabText(tab_type);
+    return ui->tabWidget->tabText(int(tab_type));
 }
 
 QKeySequence RPCConsole::tabShortcut(TabTypes tab_type) const
 {
     switch (tab_type) {
-    case TAB_INFO: return QKeySequence(Qt::CTRL + Qt::Key_I);
-    case TAB_CONSOLE: return QKeySequence(Qt::CTRL + Qt::Key_T);
-    case TAB_GRAPH: return QKeySequence(Qt::CTRL + Qt::Key_N);
-    case TAB_PEERS: return QKeySequence(Qt::CTRL + Qt::Key_P);
+    case TabTypes::INFO: return QKeySequence(Qt::CTRL + Qt::Key_I);
+    case TabTypes::CONSOLE: return QKeySequence(Qt::CTRL + Qt::Key_T);
+    case TabTypes::GRAPH: return QKeySequence(Qt::CTRL + Qt::Key_N);
+    case TabTypes::PEERS: return QKeySequence(Qt::CTRL + Qt::Key_P);
     }
+    return QKeySequence();
```

Comment:
This code should never execute

```suggestion
    assert(false);
```

Reply:
There's only warnings for unhandled cases:
```
qt/rpcconsole.cpp:1281:13: warning: enumeration value 'INFO' not handled in switch [-Wswitch]
    switch (tab_type) {
            ^
qt/rpcconsole.cpp:1281:13: note: add missing switch cases
    switch (tab_type) {
            ^
1 warning generated.
```

Reply:
This is a compiler error, i.e. the compilation will not succeed. See https://github.com/bitcoin/bitcoin/pull/16424#issuecomment-513354529

Reply:
I think we need more documentation around this. I am repeating the same points every couple of weeks.

Reply:
Oh sorry then (have to check why it doesn't fail here). I'll change as you suggest.

Reply:
Have you done `--enable-werror`?

Reply:
No, that wasn't set, thanks 👍